gpii/node_modules/eventLog/package.json

```diff
@@ -0,0 +1,13 @@
+{
+    "name": "eventLog",
+    "description": "Event logging.",
+    "version": "0.3.0",
```

Comment:
We'll deal with all of these issues (including prefixing every module name with "gpii-", etc.) in some future GIANT PURGE (TM) - right now, we may as well be consistent